tests/certbot-pebble-integration.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
(Copied from last review because it's easy to miss full-review comments)

Larger question: why is Pebble integration in this PR? Were you using it to test against by commenting out parts of certbot-boulder-integration? If we're going to include it, let's put something helpful there, like some test that does run properly.

Reply:
Gotcha. Seems good either way.